### PR TITLE
New multi contacts on actioncomm

### PR DESCRIPTION
**Pull request by aconits**
# New possibility to link 1 or more contact to event
Use the llx_actioncomm_resources table to save the link with socpeople element
I add a function in the migration step to transfer the fk_contact from llx_actioncomm to llx_actioncomm_resources
I keep the save about fk_contact with the first contact

### CODE REVIEW

**Comment by ghost** [comment]
@eldy Not interrested on this feature ?

File: htdocs/comm/action/class/actioncomm.class.php [hunk 1]
```diff
@@ -600,6 +625,52 @@ function fetch($id, $ref='',$ref_ext='')
...
+            {
+            	if ($obj->fk_element > 0)
+				{
+					switch ($obj->element_type) {
+						case 'user':
+							$this->userassigned[$obj->fk_element]=array('id'=>$obj->fk_element, 'mandatory'=>$obj->mandatory, 'answer_status'=>$obj->answer_status, 'transparency'=>$obj->transparency);
+							if (empty($this->userownerid)) $this->userownerid=$obj->fk_element;	// If not defined (should not happened, we fix this)
+							break;
+						case 'socpeople':
+							$this->TContactId[] = $obj->fk_element;
```
Comment (eldy) [inline htdocs/comm/action/class/actioncomm.class.php]: Why setting an array $this->TContactId ? 
There is already array $this->socpeoplaassigned and 
$this->TContactId    seems same than    array_keys($this->socpeopleassigned)

**Comment by eldy** [comment]
ping @atm-florian @atm-ph

**Comment by aconits** [comment]
I don't remember why I use a new attribute instead socpeopleassigned
I'll look this week to give you a feedback

**Comment by eldy** [comment]
@atm-ph I will freeze version 7.0 soon. If you can just fix this (remove $this->TContactId and use array_keys($this->socpeopleassigned) where you need the array), I would be able to include it in 7.0

**Pull request merged**